.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @exercism/maintainers-admin
+.github/CODEOWNERS @exercism/maintainers-admin
```

Comment:
Could you revert this change? We purposefully want an admin to sign off on any tooling.